src/modules/storage/LedgerStorage.ts

```diff
@@ -2153,6 +2223,25 @@ export class LedgerStorage extends Storages
         return this.query(sql, []);
     }
 
+    /**
+     * Get all Average Fees
+     */
+    public getAvgFee (): Promise<any[]>
+    {
+        const sql = `SELECT fee FROM fees
+            ORDER BY time_stamp DESC`
+        return this.query(sql, []);
+    }
```

Comment:
There may be an average fee calculation formula.
For example, the average fee of the last 100 blocks can be calculated.